### PR TITLE
chore: ignore .DS_Store files

### PR DESCRIPTION
**Pull request by levibostian**
Complete each step to get your pull request merged in. [Learn more about the workflow this project uses](https://github.com/customerio/customerio-ios/blob/develop/docs/dev-notes/GIT-WORKFLOW.md). 
- [ ] [Assign members of your team](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to review the pull request. 
- [ ] Wait for pull request status checks to complete. If there are problems, fix them until you see that [all status checks are passing](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fsymfony.com%2Fdoc%2F4.3%2F_images%2Fdocs-pull-request-symfonycloud.png&f=1&nofb=1). 
- [ ] Wait until the pull request has been reviewed *and approved* by a teammate
- [ ] After pull request is approved, and you determine it's ready **add the label "Ready to merge"** to the pull request. A bot will *squash and merge* the pull request for you after the label is added.

### CODE REVIEW

File: Sources/Common/Background Queue/Queue.swift [hunk 1]
```diff
@@ -123,11 +123,17 @@ public class CioQueue: Queue {
...
+            type: QueueTaskType.trackDeliveryMetric.rawValue,
+            data: TrackDeliveryEventRequestBody(
+                type: .inApp,
+                payload: DeliveryPayload(
+                    deliveryId: deliveryId,
+                    event: event,
+                    timestamp: dateUtil.now
+                )
+            )
+        )
```
Comment (matt-frizzell) [inline Sources/Common/Background Queue/Queue.swift:136]: Was this style change an automatic linting deal or is this an update to personal/squad code-style? It doesn't look relevant to this push (other than making things nicer as you're here), so I figured I would ask.

File: Sources/MessagingInApp/autogenerated/AutoLenses.generated.swift [hunk 2]
```diff
@@ -46,8 +46,10 @@ struct Lens<Whole, Part> {
 }
 
 func * <A, B, C>(lhs: Lens<A, B>, rhs: Lens<B, C>) -> Lens<A, C> {
-    Lens<A, C>(get: { a in rhs.get(lhs.get(a)) },
-               set: { c, a in lhs.set(rhs.set(c, lhs.get(a)), a) })
+    Lens<A, C>(
+        get: { a in rhs.get(lhs.get(a)) },
+        set: { c, a in lhs.set(rhs.set(c, lhs.get(a)), a) }
```
Comment (matt-frizzell) [inline Sources/MessagingInApp/autogenerated/AutoLenses.generated.swift:51]: Do we care about the code coverage warning? In this case it's just being triggered by style changes, but I guess my question is, should this be covered? if so, when might we come back to this? Are we or should we track places to cover with tests as part of a clean-up activity?

**Review by matt-frizzell — APPROVED** [review]
Approved. All the changes that matter to ignoring DS Store are good. Added comments are more informational than functional for this PR.

File: Sources/Common/Background Queue/Queue.swift (hunk 1 above)
Comment (levibostian) [reply Sources/Common/Background Queue/Queue.swift:136]: I did make a commit in this PR for formatting swift code. We use git hooks and a tool (swiftformat) that formats code the same for everyone on the team, automatically. 

However, if a developer on our team doesn't have git hooks installed, code formatting does not automatically happen for them. Which introduces unformatted code to the codebase. This is what happened here which is why I introduced a commit to run the formatter.

File: Sources/MessagingInApp/autogenerated/AutoLenses.generated.swift (hunk 2 above)
Comment (levibostian) [reply Sources/MessagingInApp/autogenerated/AutoLenses.generated.swift:51]: This particular file that you are talking about is generated for us. We tend to ignore those files entirely in code review because it's automatically generated. So, test coverage on this code isn't super important, IMO.
